test/user_test.rb

```diff
@@ -0,0 +1,51 @@
+require 'minitest/autorun'
+require 'minitest/pride'
+require './lib/user'
+require './lib/joke'
+
+class UserTest < Minitest::Test
+  def setup
+    @joke_1 = Joke.new(1, "Why did the strawberry cross the road?", "Because his mother was in a jam.")
```

Comment:
It would have been ideal to use an `@sal` variable in the setup method along with the two jokes. This would speed up performance!